src/devices/Max31865/Max31865.cs

```diff
@@ -0,0 +1,290 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.Spi;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Max31865
+{
+    /// <summary>
+    /// MAX31865 RTD-to-Digital Converter
+    /// </summary>
+    public class Max31865 : IDisposable
+    {
+        private const short FilterMode60HzSleep = 55;
+        private const short FilterMode50HzSleep = 65;
+
+        // Callender Van Dusen coefficiants A/B for temperature conversion
+        private const double RTD_A = 3.9083e-3;
```

Comment:
Maybe better to use a normal naming like `TemperatureCoefficiantA` and B?
Also what RTD stands for? It appear few times and I think I didn't see a proper long name for this abbreviation. You use the abbreviation in the /// comments, so would be good to have it plain if possible in anything public comment

Reply:
The enum ResistanceTemperatureDetectorWires contains the proper name for RTD Resistance Temperature Detector. I can change RTD to the proper name if required